src/Log.hpp

```diff
@@ -38,8 +41,11 @@ class LoggerStream : public std::ostringstream
 
 extern Logger g_nLog;
 
+#define NOTIFIER_LOGGER usbguardNotifier::g_nLog
```

Comment:
If possible I would prefer not using define here and maybe use something else... maybe keyword "using" might do the trick

Reply:
**using** and **define** are not the same. Defines are not program statements but directives for the preprocessor.

Reply:
I teach C programming you dont have to tell me that Xd . I just dislike using defines in C++ when there are better options for sure

Reply:
usbguardNotifier::g_nLog is not a type, so in my opinion you can not use ``using`` for this purpose. However, if you have a better solution for this, I would like to know about that. Thanks.